.github/workflows/build_and_run.yml

```diff
@@ -71,7 +71,7 @@ jobs:
 
           dpbench \
           -i python,numpy,dpnp,sycl,numba_n,numba_np,numba_npr,numba_dpex_k,numba_dpex_n,numba_dpex_p \
-          run -r2 --no-print-results
```

Comment:
Why no validate? For the CI runs we do want to validate for the `s` preset. Otherwise we will end up with broken implementations getting pushed.

Reply:
I see. Ok in that case.